packages/reactive_ble_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.3
+* Enable extended advertising on android. Fix #571
```

Comment:
Shall I suggest you a Markdown linter? :)

Reply:
Yeah I think so :smile:

Reply:
I have this one installed (in VS Code): https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint